### PR TITLE
Remove images from the website

### PR DESCRIPTION
**Pull request by manuelafm**
null

----

Site preview: https://igalia.github.io/wpewebkit.org/mafm-remove-images/

### CODE REVIEW

**Review by mariospr — CHANGES_REQUESTED** [review]
This is a good cleanup, but I think we need to include in this PR changes in any other places where they reference the Balena blocks, the yocto images and flatpak.

From what I can see, and besides what's included in his PR already, those other places are these:

  * https://wpewebkit.org/developers -> We should remove the 3 links pointing to flatpak, yocto and Balena in the WPE Builds section; and probably we'd like to reword the paragraph there to include the link to the "Linux distros" link in a more natural way now that we only have one item left.

  * https://wpewebkit.org/blog/04-wpe-qa-tooling.html -> I know it's a blog post, but still I think we should reword the "Testing WPE in the wild" section to only include the links to the "prebuild distro packages"; and again I would reword the section a bit now that we only have one item (and remove the last sentence i.d. "Some of these options...").

  * https://wpewebkit.org/blog/02-overview-of-wpe.html -> The last paragraph reference to the flatpak and balena pages too, so we need to do some rewording there to remove those as well.

  * https://wpewebkit.org/about/supported-hardware.html -> There is a link to "Try WPE with Flatpak" from the "Resources" section at the bottom. We need to remove that and replace it with something else; I'd suggest maybe linking to the [Blog](https://wpewebkit.org/blog/) section of the website, with an excerpt of the text at the top as preview text.

And I think that's it, or at least I could not find more pointers to those dead sections.

**Comment by manuelafm** [comment]
> This is a good cleanup, but I think we need to include in this PR changes in any other places where they reference the Balena blocks, the yocto images and flatpak.
> 
> From what I can see, and besides what's included in his PR already, those other places are these:
> 
>     * https://wpewebkit.org/developers -> We should remove the 3 links pointing to flatpak, yocto and Balena in the WPE Builds section; and probably we'd like to reword the paragraph there to include the link to the "Linux distros" link in a more natural way now that we only have one item left.

Done.

>     * https://wpewebkit.org/blog/04-wpe-qa-tooling.html -> I know it's a blog post, but still I think we should reword the "Testing WPE in the wild" section to only include the links to the "prebuild distro packages"; and again I would reword the section a bit now that we only have one item (and remove the last sentence i.d. "Some of these options...").
> 
>     * https://wpewebkit.org/blog/02-overview-of-wpe.html -> The last paragraph reference to the flatpak and balena pages too, so we need to do some rewording there to remove those as well.

I am not sure that it is a good idea to change blog posts in general, but given that these are more like articles about WPE that we control instead of typical blog posts, I suppose that it's all right.

I tried to keep it as simple as possible by using a "greedy removing algorithm" and not trying to rephrase.

>     * https://wpewebkit.org/about/supported-hardware.html -> There is a link to "Try WPE with Flatpak" from the "Resources" section at the bottom. We need to remove that and replace it with something else; I'd suggest maybe linking to the [Blog](https://wpewebkit.org/blog/) section of the website, with an excerpt of the text at the top as preview text.

I think that removing it is fine?  The previous resource is the same "Get Started" already contains the rest, including containing Flatpak.
 
> And I think that's it, or at least I could not find more pointers to those dead sections.

I just pushed another change, but since I'm not going to be around the next week in general, please feel free to go ahead and make further changes.

File: developers/index.md [hunk 1]
```diff
@@ -33,12 +33,7 @@ h2 {
 
 While there are several simple ways for developers to experiment with and explore WPE, none are tuned for performance. Generally, shipping products for embedded systems are performance-tuned custom builds. To make this easier, there is also [meta-webkit](https://github.com/Igalia/meta-webkit), which provides build recipes, WebKit based runtimes, and browsers for use with OpenEmbedded and/or Yocto.
 
-<ul class="arrows">
-<li><a href="{{ '/about/explore-wpe.html' | url }}">Linux distros for Debian, Ubuntu, Raspbian, Arch, Fedora...</a></li>
-<li><a href="{{ '/about/flatpak.html' | url }}">Install WPE and minibrowser with flatpak</a></li>
-<li><a href="{{ '/about/rpi-yocto-wpe.html' | url }}">Install with images for Raspberry Pi</a></li>
-<li><a href="{{ '/about/balena-wpe.html' | url }}">Install with Balena</a></li>
-</ul>
+There are also pre-build packages available for many <a href="{{ '/about/explore-wpe.html' | url }}">Linux distros</a>.
```
Comment (aperezdc) [inline developers/index.md:36]: Let's use formal wording in a formal website: `s/distro/distribution/`&mdash;also for consistency with the other occurrences 😉 

```suggestion
There are also pre-build packages available for many <a href="{{ '/about/explore-wpe.html' | url }}">Linux distributions</a>.
```

**Review by aperezdc — CHANGES_REQUESTED** [review]
Patch LGTM overall, I have only a small nit, please check the comment.

**Review by mariospr — COMMENTED** [review]
I agree with the changes proposed by @aperezdc and also with the latest changes done by @manuelafm following my previous comments, but at this point I'd like to get the feedback from either @bkardell or @meyerweb since there are some changes in the wording when removing those lists, that might need some massaging :innocent:

**Comment by manuelafm** [comment]
@bkardell or @meyerweb , feedback please?

**Comment by meyerweb** [comment]
> @bkardell or @meyerweb , feedback please?

Looks good to me, at least from an editing point of view.  (No idea if the changes are technically accurate or not, so can’t comment on that part.)

**Review by mariospr — APPROVED** [review]
LGTM now, thanks
